agent/php_execute.c

```diff
@@ -1950,26 +1953,21 @@ void nr_php_observer_fcall_end(zend_execute_data* execute_data,
     return;
   }
 
-  NRPRG(php_cur_stack_depth) -= 1;
-
-  if (nrunlikely(0 == nr_php_recording(TSRMLS_C))) {
-    return;
-  }
-
-  int show_executes
-      = NR_PHP_PROCESS_GLOBALS(special_flags).show_executes
-        || NR_PHP_PROCESS_GLOBALS(special_flags).show_execute_returns;
-
-  if (nrunlikely(show_executes)) {
+  if (nrlikely(1 == nr_php_recording())) {
+    int show_executes_return
+        = NR_PHP_PROCESS_GLOBALS(special_flags).show_execute_returns;
     /*
      * For OAPI don't call nr_php_execute_enabled from execute_show.
-     * nr_php_execute_show updated in another ticket.
      * Can show execute and returns here.
```

Comment:
I don't think this comment is needed anymore.

Reply:
I liked the idea of keeping them in place as an explanation for why it was done this way as opposed to the old paradigm

Reply:
IMHO it only adds confusion as to why a search for `nr_php_execute_enabled` would find the oapi code where `nr_php_execute_enabled` is not used and expected to appear. `nr_php_execute_enabled` is not oapi way of instrumenting that's been replaced with `nr_php_observer_fcall_begin` and `nr_php_observer_fcall_end`.  Things are done differently in oapi. This comment made sense when `nr_php_observer_fcall_begin` and `nr_php_observer_fcall_end` were just a stubs to indicate what should be implemented in them. Now that the implementation is in place, there's no longer need for this comment.

Reply:
Can you add a suggestion comment to both so I can commit that change here directly?

Reply:
Unfortunately GH's code review suggestions cannot be made to code that was not modified in PR. So I cannot create suggestion for this comment.

Reply:
Updated and pushed